src/ResourceManager/ServiceBus/Commands.ServiceBus/Utilities/ServiceBusClient.cs

```diff
@@ -646,6 +735,54 @@ public IEnumerable<PSSharedAccessAuthorizationRuleAttributes> ListAliasAuthoriza
 
 
         #endregion
+        
+
+        #region MigrationConfiguration
+        public PSServiceBusMigrationConfigurationAttributes GetServiceBusMigrationConfiguration(string resourceGroupName, string namespaceName)
+        {
+            var response = Client.MigrationConfigs.Get(resourceGroupName, namespaceName);
+            return new PSServiceBusMigrationConfigurationAttributes(response);
+        }
+
+        public PSServiceBusMigrationConfigurationAttributes StartServiceBusMigrationConfiguration(string resourceGroupName, string namespaceName, PSServiceBusMigrationConfigurationAttributes parameter)
+        {
+            var Parameter1 = new Management.ServiceBus.Models.MigrationConfigProperties();
+
+            if (!string.IsNullOrEmpty(parameter.PostMigrationName))
+                Parameter1.PostMigrationName = parameter.PostMigrationName;
+
+            if (!string.IsNullOrEmpty(parameter.TargetNamespace))
+                Parameter1.TargetNamespace = parameter.TargetNamespace;
+
+            var response = Client.MigrationConfigs.BeginCreateAndStartMigration(resourceGroupName, namespaceName, Parameter1);
+            return new PSServiceBusMigrationConfigurationAttributes(response);
+        }
+
+        public bool DeleteServiceBusMigrationConfiguration(string resourceGroupName, string namespaceName)
+        {
+            Client.MigrationConfigs.Revert(resourceGroupName, namespaceName);
+            Thread.Sleep(TimeSpan.FromSeconds(5));
```

Comment:
@v-Ajnava for the few instances of `Thread.Sleep` in this file, please use `TestMockSupport.Delay(5000)` instead so tests in playback aren't waiting

Reply:
Resolved